subgenerator/templates/index.js

```diff
@@ -20,19 +19,30 @@ module.exports = yeoman.Base.extend({
       default: true
     }];
 
-    this.prompt(prompts, function (props) {
-      this.props = props;
+    return this.prompt(prompts).then(function (props) {
       // To access props later use this.props.someAnswer;
+      this.props = props;
 
-      done();
     }.bind(this));
   },
 
   writing: function () {
+    //Simple file copy example
```

Comment:
Let's remove these new comments. I don't think they're not adding any value to the end users.